src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/EqualsTest.java

```diff
@@ -0,0 +1,367 @@
+            void longFiltersWithDifferentFilterTypesDoubleFilter(int id, String name, String description, String productAttribute, long min, long max) {
+                LongFilter longFilter = new LongFilter(id, name, description, productAttribute, min, max);
+                DoubleFilter doubleFilterWithDifferentType = new DoubleFilter(id, name, description, productAttribute, (double) min, (double) max);
+                assertNotEquals(longFilter,doubleFilterWithDifferentType);
+            }
+
+            @ParameterizedTest(name = "Time filter: {0},{1},{2},{3},{4},{5}")
+            @DisplayName("Long filters with different filter types: Time filter")
+            @CsvFileSource(resources = "LongFilter.csv", numLinesToSkip = 1)
+            void longFiltersWithDifferentFilterTypesTimeFilter(int id, String name, String description, String productAttribute, long min, long max) {
+                LongFilter longFilter = new LongFilter(id, name, description, productAttribute, min, max);
+                TimeFilter timeFilterWithDifferentType = new TimeFilter(id, name, description, productAttribute, Instant.ofEpochSecond(min), Instant.ofEpochSecond(max));
+                assertNotEquals(longFilter,timeFilterWithDifferentType);
+            }
+
+            @ParameterizedTest(name = "Different user min: {0},{1},{2},{3},{4},{5}")
+            @DisplayName("Long filters with different UserMin values")
+            @CsvFileSource(resources = "LongFilter.csv", numLinesToSkip = 1)
+            void longFiltersWithDifferentUserMinValues(int id, String name, String description, String productAttribute, long min, long max) throws IllegalMinMaxException {
+                LongFilter longFilter = new LongFilter(id, name, description, productAttribute, min, max);
+                LongFilter longFilterWithDifferentUserMin = new LongFilter(id, name, description, productAttribute, min, max);
+                longFilterWithDifferentUserMin.setUserMin(min+1);
+                assertNotEquals(longFilter,longFilterWithDifferentUserMin);
+            }
+
+            @ParameterizedTest(name = "Different user max: {0},{1},{2},{3},{4},{5} ")
+            @DisplayName("Long filters with different UserMax values")
+            @CsvFileSource(resources = "LongFilter.csv", numLinesToSkip = 1)
+            void longFiltersWithDifferentUserMaxValues(int id, String name, String description, String productAttribute, long min, long max) throws IllegalMinMaxException {
+                LongFilter longFilter = new LongFilter(id, name, description, productAttribute, min, max);
+                LongFilter longFilterWithDifferentUserMax = new LongFilter(id, name, description, productAttribute, min, max);
+                longFilterWithDifferentUserMax.setUserMax(max-1);
+                assertNotEquals(longFilter,longFilterWithDifferentUserMax);
+            }
+        }
+
+        @Nested
+        @DisplayName("Non-matching time filter test")
```

Comment:
Same as the naming concern for double filter

Reply:
Done